controller/api/public/stat_summary_test.go

```diff
@@ -642,4 +643,135 @@ status:
 			}
 		}
 	})
+
+	t.Run("Return empty stats summary response", func(t *testing.T) {
+		t.Run("when pod phase is succeeded or failed", func(t *testing.T) {
+			expectations := []statSumExpected{
+				statSumExpected{
+					err: nil,
+					k8sConfigs: []string{`
+apiVersion: v1
+kind: Pod
+metadata:
+  name: emojivoto-00
+  namespace: emojivoto
+  labels:
+    app: emoji-svc
+  annotations:
+    conduit.io/proxy-version: testinjectversion
+status:
+  phase: succeeded
```

Comment:
Fixed! Good catch.

IIUC, the reason why these tests didn't fail was because the stat summary tests asserted the correctness of the responses, not if the pods were returned.